.packit.yml

```diff
@@ -2,6 +2,21 @@ specfile_path: anaconda.spec
 upstream_package_name: anaconda
 upstream_tag_template: anaconda-{version}-1
 copy_upstream_release_description: true
+srpm_build_deps:
+- automake
```

Comment:
This doesn't look like a consistent indentation with the rest of the file.

Reply:
fixed

(I'm used to code formatters fixing this for me :D)